### ISSUE TITLE
Location File is not created

### ISSUE THREAD

**Issue by gpires** [PokemonGoF/PokemonGo-Bot#982]
### Actual Behavior
```
127.0.0.1 - - [26/Jul/2016 13:26:12] code 404, message File not found
127.0.0.1 - - [26/Jul/2016 13:26:12] "GET /location-myactual@account.json HTTP/1.1" 404 -
```

### Other Information
OS: Win10
Git Commit: 3aca6d798f8a3e173dce0beb7b13c9505cc5b251

**Comment by ironbishop** [comment]
duplicate of #918 #905 , about to be fixed in #908 (not merged atm)

**Comment by gpires** [comment]
@ironbishop: Next time I'll check other issues. Thanks
